packages/webdriver/protocol/jsonwp.json

```diff
@@ -710,7 +710,7 @@
         "name": "args",
         "type": "(string|object|number|boolean|undefined)[]",
         "description": "the script arguments",
-        "required": true
+        "required": false
```

Comment:
```suggestion
```

Reply:
@christian-bromann I found 73 usages of `"required: false"` in all 5 json files. I supposed the idea is to specify if property is required or not explicitly. 
Don't mind removing it if I'm wrong, double checking.
Thanks

Reply:
ah ok, I wasn't sure anymore. It is fine then.